crates/routes/Cargo.toml

```diff
@@ -27,7 +27,8 @@ anyhow = "1.0.43"
 chrono = { version = "0.4.19", features = ["serde"] }
 rss = "1.10.0"
 serde = { version = "1.0.127", features = ["derive"] }
-awc = { version = "3.0.0-beta.7", default-features = false }
+serde_json = "1.0.66"
+awc = "3.0.0-beta.7"
```

Comment:
This is gonna increase our build time again, because brotli and zstd take a significant amount of time.

@asonix Any chance that you would disable gzip in pictrs instead? Imo that should be left to nginx, and it would also speed up pictrs build. But its annoying that you'd have to publish a new version.

Reply:
I can leave a todo in the toml there.